packages/timeline-state-resolver/src/integrations/httpSend/index.ts

```diff
@@ -199,6 +200,23 @@ export class HTTPSendDevice
 				headers: command.content.headers,
 			}
 
+			const url = new URL(command.content.url)
+			if (!this.options.noProxy?.includes(url.host)) {
+				if (url.protocol === 'http:' && this.options.httpProxy) {
+					options.agent = {
+						http: new HttpProxyAgent({
```

Comment:
I did wonder that but I couldn't decide from the documentation if it was safe to reuse the agent for each request and if so do you need to provide some sensible config options to do it.
All the examples I could find show a new agent being created every time, but that could simply be because they're smalls examples.

I can add the destroy if you think it's worth doing?